SwitchItem/SwitchItem.js

```diff
@@ -88,6 +98,7 @@ const SwitchItemBase = kind({
 	},
 
 	defaultProps: {
+		disabled: false,
 		offText: 'off',
```

Comment:
Think we should `$L` these strings since we're in here already?

Reply:
They wouldn't get updated on locale change if we did it here.